src/API/Endpoints/Reports/Endpoint.php

```diff
@@ -142,6 +155,18 @@ public function sanitize_date( $param, $request, $key ) {
 		return $date;
 	}
 
+	public function validate_currency( $param, $request, $key ) {
+		return in_array( $param, array_keys( give_get_currencies_list() ) );
+	}
+
+	public function validate_test_mode( $param, $request, $key ) {
+		return $param === 'true' || $param === 'false';
+	}
+
+	public function sanitize_test_mode( $param, $request, $key ) {
+		return $param === 'true' ? true : false;
```

Comment:
@henryholtgeerts You can use `filter_var`.

```suggestion
		return filter_var($param, FILTER_VALIDATE_BOOLEAN);
```